src/net_processing.h

```diff
@@ -64,6 +64,8 @@ struct CNodeStateStats {
     int nSyncHeight;
     int nCommonHeight;
     std::vector<int> vHeightInFlight;
+    uint64_t m_addr_processed = 0;
+    uint64_t m_addr_rate_limited = 0;
```

Comment:
Those two new fields are never updated to the corresponding one in CNode

Reply:
You're right this was not supposed to be included.

Reply:
You are using it in the RPC, don't remove it but update the value somewhere

Reply:
They are now updating